doc/nrf/releases/release-notes-latest.rst

```diff
@@ -436,17 +462,26 @@ The following list summarizes the most important changes inherited from upstream
       * :option:`CONFIG_OPENTHREAD_MAC_SOFTWARE_RETRANSMIT_ENABLE`
       * :option:`CONFIG_OPENTHREAD_PLATFORM_USEC_TIMER_ENABLE`
       * :option:`CONFIG_OPENTHREAD_CONFIG_PLATFORM_INFO`
+      * :option:`OPENTHREAD_RADIO_LINK_IEEE_802_15_4_ENABLE`
+      * :option:`OPENTHREAD_RADIO_LINK_TREL_ENABLE`
+      * :option:`OPENTHREAD_CSL_SAMPLE_WINDOW`
+      * :option:`OPENTHREAD_CSL_RECEIVE_TIME_AHEAD`
+      * :option:`OPENTHREAD_MAC_SOFTWARE_CSMA_BACKOFF_ENABLE`
```

Comment:
Missed that these all need to start with CONFIG_ - sorry. This will fail in the doc build.

Reply:
Indeed, will fix rightaway.